packages/celotool/src/e2e-tests/utils.ts

```diff
@@ -438,6 +436,29 @@ export async function initAndStartGeth(gethBinaryPath: string, instance: GethIns
   return startGeth(gethBinaryPath, instance)
 }
 
+// Add each validator as a peer of each other validator via the admin interface.
+async function connectValidatorPeers(gethConfig: GethTestConfig) {
```

Comment:
Instead of connecting each validator to the other, can we connect them all to validator 0? This allows us to test that the validators still are able to peer with each other.

Reply:
Done